tests/unit/admin/test-class-wc-rest-payments-settings-controller.php

```diff
@@ -255,10 +267,34 @@ public function test_get_settings_fails_if_user_cannot_manage_woocommerce() {
 		remove_filter( 'user_has_cap', $cb );
 	}
 
+	public function test_get_settings_without_error_when_faulty_enabled_payment_methods() {
+		$this->gateway->update_option(
+			'available_payment_method_ids',
+			[
+				Payment_Method::CARD,
```

Comment:
I know other tests are not following my feedback, but I suggest keeping only `card` and `sepa` here - other methods are not making sense in the context of this test case. When I'm reading it I'm not getting why all of those are saved - reducing the list will help the reader to understand the intention much easier.

Reply:
That makes sense. For the original one, I had taken a clue from another test, and to mock a common situation where most PM's ( excluding Link ) are enabled. But now that I think, they are not all needed, except one additional method other than card. Thanks. 

Made the change here - 88cc6c1

Marking as resolved.